registry/registry.go

```diff
@@ -317,6 +328,11 @@ func alive(path string, handler http.Handler) http.Handler {
 
 		handler.ServeHTTP(w, r)
 	})
+
+	httpMetrics := prometheus.RootHTTPNamespace.NewDefaultHttpMetrics("root")
```

Comment:
I don't think this counter is necessary: it will count for all requests. This can just be rolled up from the other counters.